### DISCUSSION TITLE
Any plans to update and continue this project?

### DISCUSSION THREAD

**Discussion by bynarie** [Open-Shell/Open-Shell-Menu#1961] [General]
Seems like the last release was almost a year ago.. It's still working great but are there any plans to continue development into the next release of windows and/or add new features?

**Comment by ge0rdi** [comment]
The project is mostly being just maintained. There is no plan for any _bigger_ development.

I'm trying to fix eventual _breaking_ issues that prevent use of `Open-Shell` (especially on new Windows).

**Reply by moincetti** [reply]
I can't tell you how much I appreciate you guys. Default Windows start/taskbar just feels so hard to organize in comparison.

**Reply by bynarie** [reply]
I agree @moincetti. These guys are freaking awesome
